frontend/portals/ProductItemPrice/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'glamor';
+import { I18n } from '@shopgate/engage/components';
+import connect from './connector';
+
+const pointPriceStyle = css({
+  lineHeight: 1.75,
+});
+
+/**
+ * ProductItemPrice component
+ * @param {Object} redeemProduct Swell redeem option type product
+ * @param {Node} children Portal children
+ * @return {JSX|Node}
+ * @constructor
+ */
+const ProductItemPrice = ({ redeemProduct, children }) => {
```

Comment:
this component will also replace the price for all other places inside the app where this product is shown. Shouldn't it only show it for the RedeemProductsSlider?

Reply:
I thought about this. My assumption is that a product associated with the product redemption option can only be a redemption product so in any product list it is shown in should only show the points price. So no, it shouldn't only show in the RedeemProductSlider if my assumption is correct.

Reply:
ok